queries/python/highlights.scm

```diff
@@ -195,17 +195,20 @@
   "del"
 ] @keyword.operator
 
+[
+  "def"
+  "lambda"
+] @keyword.function
+
 [
   "assert"
   "async"
   "await"
   "class"
-  "def"
   "except"
   "exec"
   "finally"
   "global"
-  "lambda"
```

Comment:
The original intention of `@keyword.function` was to resemble the style of Vim regex highlights for `end`-based languages. They usually have the function highlight applied to the `end` block while other languages use regular `keyword` for `func` `def`
![image](https://user-images.githubusercontent.com/7189118/123321562-69cca600-d533-11eb-8455-76a11f3d50ca.png)

![image](https://user-images.githubusercontent.com/7189118/123321741-ac8e7e00-d533-11eb-9343-b8ca1a0449a7.png)